edward/inferences/inference.py

```diff
@@ -151,6 +151,13 @@ def __init__(self, latent_vars=None, data=None, model_wrapper=None):
                               "shape.")
 
             self.data[key] = value
+          elif isinstance(value, np.float64):
```

Comment:
cool. is there a way to get this work for non-float64 types too? similar to [`L137-140`](https://github.com/nfoti/edward/blob/2970e4b71270225dcd5596d38e4bab646470b38d/edward/inferences/inference.py#L137-L140), so that you can enter into this clause if it's int or float.